### PR TITLE
Unit & Integration Testing ; MainPage

### PR DESCRIPTION
**Pull request by BrigetteDoelp**
## Pull request type
Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Other (please describe):
Testing for `MainPage` component is completed! 
## Problem/feature
Unit and integration tests are needed
## Solution
Unit and integration tests have been written.
Revolves #65 
Revoles #64 
## Other changes (e.g. bug fixes, UI tweaks, small refactors)
Fixed a type in `CallToAction.js` on line 8.
## Checklist
- [x] Code compiles correctly
- [x] Tests for the changes have been added (for bug fixes/features)
- [x] All tests passing
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: src/components/CallToAction/CallToAction.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ const CallToAction = (props) => {
   return (
     <section className ='CallToAction'>
       <h1>{props.title || "Looks like you aren't signed in yet!"}</h1>
-      <p> Sign in if you already have an acount with us,
+      <p> Sign in if you already have an account with us,
```
Comment (nathanielmillard) [inline src/components/CallToAction/CallToAction.js:8]: Thanks for catching this!

**Review by nathanielmillard — APPROVED** [review]
This is looking great 👍

**Comment by BrigetteDoelp** [comment]
> This is looking great 👍

Thank you! Also just wanted to confirm before moving forward, we're not testing that the `Link` bits work, yes?

**Comment by nathanielmillard** [comment]
> > This is looking great 👍
> 
> Thank you! Also just wanted to confirm before moving forward, we're not testing that the `Link` bits work, yes?

We should for larger inegrations and acceptance tests on App
As if to say as a user when I click on a Link I should see the render change form one thing to another.

**Comment by BrigetteDoelp** [comment]
> > > This is looking great 👍
> > 
> > 
> > Thank you! Also just wanted to confirm before moving forward, we're not testing that the `Link` bits work, yes?
> 
> We should for larger inegrations and acceptance tests on App
> As if to say as a user when I click on a Link I should see the render change form one thing to another.

Awesome, thank you! I'll work on getting testing started for the rest of the components for now and revisit that after we have a good base going.

**Review by DougieDev — APPROVED** [review]

**Pull request merged**